### ISSUE TITLE
Windows Hello support for authentication

### ISSUE THREAD

**Issue by whiteLine1** [microsoft/WSA#106]
### Is your feature request related to a problem? Please describe

When accessing an application that uses passwords, you always have to access it by typing it.

### Describe the solution you'd like

Unlock applications such as banking with the fingerprint reader, facial identification, the use of security keys.

### Please specify the version of Windows Subsystem for Android

2208.40000.5.0

**Comment by hamza-usmani** [comment]
Hi, thank you for this feature request. Currently this isn't on our immediate roadmap, but we're always listening to the community!

**Comment by andrewleader** [comment]
Hi @whiteLine1, can you share the names of the apps you develop that use biometric authentication?

**Comment by s1204IT** [comment]
To be precise, you can set the screen lock, but you can not unlock it when WSA boot.

**Comment by s1204IT** [comment]
> > To be precise, you can set the screen lock, but you can not unlock it when WSA boot.
> 
> how can I set screen lock other than "swipe"?

https://github.com/LSPosed/MagiskOnWSALocal/discussions/172#discussioncomment-3830977
Please understand the risks.

https://github.com/microsoft/WSA/issues/172
You need [TestDPC](https://github.com/googlesamples/android-testdpc) to make that happen.
However, `device_admin` feature is not yet implemented in WSA.
So you have to modify the image file directly or use [MagiskOnWSA(Local)](https://github.com/LSPosed/MagiskOnWSALocal).

After that, you can set the screen lock by setting **TestDPC** as the device owner from ADB and resetting the password from the application.

**Comment by rbeesley** [comment]
I was trying to do the same thing as @lulMeow. This seems like a feature which is important if Windows is going to have an Android subsystem. For one, if Microsoft were to release a future "handheld" device that is running Windows, but it seamlessly runs Android, then there needs to be a tight integration of Hello with Android's security to enable biometric. As a former Lumia user and currently owning Duo and Duo 2, I hope that this is a device that the Surface team is seriously considering, completely blurring the definition of what OS the device is running. I just want to be able to casually move from device to device and have access to all the apps I need. Having Hello integrated with WSA means that I can at least have one secure locker for passwords and authentication, and I can synchronize that between my devices.

**Comment by raaomoka** [comment]
Yes, this is another major functionality that needs to be added on WSA. As well as support for sharing content via the Windows Share Dialog to WSA apps. There's also the following feedbacks for them on the feedback hub:

https://aka.ms/AAl24lc
https://aka.ms/AAl24ll

Kindly implement these core functionalities in WSA Microsoft.

**Comment by kozseb** [comment]
how to unlock the screen because the swipe doesn't work?

**Comment by Firehawke** [comment]
This one's a bit big for me as well. My ISP's Android app allows biometric login and it'd be nice if I could use Windows Hello to authenticate.
